### ISSUE TITLE
Table broken in mails

### ISSUE THREAD

**Issue by Hiteshm01** [rohit-prajapati/html-tablify#1]
Table comes broken in mails as there is no spacing in html tags while creating table

``` html
<td>5</td></tr><tr><td>79984</td><td>4</td></tr><tr><td>82174</td><td>5</td></tr><tr><td>82525</td><td
 >2</td><
 /tr><tr><td>82775</td><td>12</td>
```

which _would_ had been perfect with spacing in tags as in 

``` html
 <td> 5 </td></tr><tr> <td> 79984 </td> <td> 4 </td></tr><tr> <td> 82174 </td> <td> 5 </td></tr><tr> <td> 82525 </td> <td>2 </td> </tr><tr> <td> 82775 </td> <td> 12 </td>
```

**Comment by rohitprajapati** [comment]
Can you please paste me the complete table html here.

Thanks

**Comment by rohitprajapati** [comment]
I think I got the issue. According to me the problem is with word wrap. As there is no space in the html so the client (in this case the Gmail) was breaking it anywhere. Will resolve it by adding new lines at the end of each row i.e. <tr>.

**Comment by rohitprajapati** [comment]
Resolved in version 1.1.1.
